indico/htdocs/js/custom/clipboard.js

```diff
@@ -0,0 +1,43 @@
+$(function() {
+    /* Show a qTip with the given text under the given element. The qTip is
+     * destroyed when hidden and thus will be shown only once. */
+    function showQTip(element, text) {
+        var container = $('<span>').qtip({
+            overwrite: true,
+            position: {
+                target: $(element)
+            },
+            content: {
+                text: text
+            },
+            show: {
+                solo: true
+            },
+            hide: {
+                event: 'unfocus click'
+            },
+            events: {
+                hide: function(evt, api) {
+                    $(this).qtip('destroy');
+                }
+            }
+        });
+        container.qtip('show');
+    }
+
+    /* Handle clicks on .js-copy-to-clipboard with cliboard.js.
+     * For simple usage, the clipboard-text data attribute will be copied to
+     * the system clipboard. For other possibilities, see https://clipboardjs.com/
+     * */
+    var c = new Clipboard('.js-copy-to-clipboard');
+    c.on('success', function(evt) {
+        showQTip(evt.trigger, $T.gettext("Copied to clipboard"));
+    });
+    c.on('error', function(evt) {
+        var copyShortcut = "CTRL-C";
+        if (/^Mac/i.test(navigator.platform)) {
+            copyShortcut = "⌘-C ";
+        }
+        showQTip(evt.trigger, $T.gettext("Press {0} now to copy").format(copyShortcut));
```

Comment:
`Press <strong>{0}</strong> to copy`? the "now" seems a bit out of place, and bold might look better on the shortcut